README.md

```diff
@@ -40,4 +47,13 @@ $ pip install -e .
 ```bash
 $ pip install '.[dev]'
 ```
-(Make sure to include the single quotes)
\ No newline at end of file
+(Make sure to include the single quotes)
+
+### Installing healpy on Mac
```

Comment:
I suggest combining this with the above section like "special care for macs".

Reply:
I like your point about including `conda install healpy` before the instructions for the pip install; looking at the healpy documentation, it does look like that will always be the route that all Mac users will need to take, so it's best to preempt the error message. 

I'm not 100% satisfied by having "Installation"/"Installation (Macs)", but I'm not sure which platforms the standard installation instructions work for, so I'm hesitating to add a "(Windows/Linux)" to the first heading. 

Definitely regretting not having taken any of those technical writing courses yet. What changes would you make to this version?

 ## Installation


```bash
$ git clone https://github.com/astronomy-commons/hipscat
$ cd hipscat
$ pip install -e .
```

## Installation (Macs)
Native prebuilt binaries for healpy on Apple Silicon Macs [do not yet exist](https://healpy.readthedocs.io/en/latest/install.html#binary-installation-with-pip-recommended-for-most-other-python-users), so it's recommended to install via conda before proceeding to hipscat.
```bash
$ conda config --add channels conda-forge
$ conda install healpy
$ git clone https://github.com/astronomy-commons/hipscat
$ cd hipscat
$ pip install -e .
``` 

#### Installing dev dependencies

```bash
$ pip install '.[dev]'
```
(Make sure to include the single quotes)

Reply:
This looks great! I don't think there's a perfect way to do it, but this gets all the necessary information across.